include/deal.II/lac/scalapack.h

```diff
@@ -189,6 +189,163 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
                const std::pair<unsigned int,unsigned int> &offset_B,
                const std::pair<unsigned int,unsigned int> &submatrix_size) const;
 
+  /**
+   * Transposing assignment: <i>A = B<sup>T</sup></i>
+   *
+   * The matrices <tt>A</tt> and <tt>B</tt> must have the same process grid.
```

Comment:
Why have you used HTML as opposed to LaTeX formatting (both here and below)?

Reply:
You should use `@p` to format the parameters of a function, i.e. `@p A` and `@p B`, when inline with other text.

Same applies below.

Reply:
I am also in favour of LaTeX, but we have a mixture of the two, apparently https://github.com/dealii/dealii/issues/5897